games-roguelike/tomenet/tomenet-4.7.3.ebuild

```diff
@@ -35,42 +36,44 @@ PATCHES=(
 
 src_prepare() {
 	default
-	use server || use dedicated || { rm -r ../lib/{config,data,save} || die ;}
+	use server || { rm -r ../lib/{config,data,save} || die ;}
```

Comment:
this is a terrible construct, turn it into a normal `if ! use server; then ... fi` construct